doc/source/tutorial/stats/discrete_yulesimon.rst

```diff
@@ -0,0 +1,41 @@
+
+.. _discrete-yulesimon:
+
+Yule-Simon Distribution
+========================
+
+A Yule-Simon random variable with parameter :math:`\alpha>0`
+can be represented as a mixture of
+exponential random variates. To see this write :math:`W` as an exponential
+random variate with rate :math:`\rho` and a Geometric random variate :math:`K`
+with probability :math:`1-exp(-W)` then :math:`K` marginally has a Yule-Simon
+distribution. The latent variable representation described above is used for
+random variate generation.
+
+.. math::
+
```

Comment:
The `math::` block below has `:nowrap:`.  This one doesn't. Perhaps it needs it?